_includes/footer.html

```diff
@@ -8,23 +8,23 @@
   {% include components/footer--big.html %}
 {% endif %}
 
-{% assign anchor = site.data.anchor %}
+{% assign identifier = site.data.usa_identifier.identifier_data[0] %}
```

Comment:
does `identifier_data` have to be an array? could we flatten it so that 

```{% assign identifier = site.data.usa_identifier```
?

Reply:
Thanks for pointing that out! I thought it was a convention, and seems like it isn't. I've update with a flatten version.